src/sentry/killswitches.py

```diff
@@ -163,6 +163,17 @@ class KillswitchInfo:
             "project_id": "A project ID to filter events by.",
         },
     ),
+    "post_process.derive-code-mappings": KillswitchInfo(
```

Comment:
How does this functionality work? Is there more documentation for this file? Is there a UI associated to modify kills switches on the fly?

Reply:
see [WOR-2359](https://getsentry.atlassian.net/browse/WOR-2359)